R/include-gld-purchases.R

```diff
@@ -31,7 +31,7 @@
 #'
 #' @examples
 #' \dontrun{
-#' register_data$lmdb |> include_gld_purchases()
+#' simulate_registers("lmdb", 100)[[1]] |> include_gld_purchases()
```

Comment:
Maybe use the name instead? 
```suggestion
#' simulate_registers("lmdb", 100)$lmdb |> include_gld_purchases()
```

Reply:
It gets annoying, and to update, its easier to do a find a replace once. We could also make individual functions to make each register... :thinking:

Reply:
I actually like that idea! Simple and clean ✅

Reply:
@signekb I'll make an issue to do that :grin: